bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -37,14 +41,61 @@ def __init__(self, bot: Bot) -> None:
         )
 
         self.reviewer = Reviewer(self.__class__.__name__, bot, self)
-        self.bot.loop.create_task(self.reviewer.reschedule_reviews())
+        self.bot.loop.create_task(self.schedule_autoreviews())
+
+    async def schedule_autoreviews(self) -> None:
+        """Reschedule reviews for active nominations if autoreview is enabled."""
+        if await self.autoreview_enabled():
+            await self.reviewer.reschedule_reviews()
+        else:
+            self.log.trace('Not scheduling reviews as autoreview is disabled.')
+
+    async def autoreview_enabled(self) -> bool:
+        """Return whether automatic posting of nomination reviews is enabled."""
+        return await self.talentpool_settings.get(AUTOREVIEW_ENABLED_KEY, True)
 
     @group(name='talentpool', aliases=('tp', 'talent', 'nomination', 'n'), invoke_without_command=True)
     @has_any_role(*MODERATION_ROLES)
     async def nomination_group(self, ctx: Context) -> None:
         """Highlights the activity of helper nominees by relaying their messages to the talent pool channel."""
         await ctx.send_help(ctx.command)
 
+    @nomination_group.group(name='autoreview', aliases=('ar',), invoke_without_command=True)
+    @has_any_role(*MODERATION_ROLES)
+    async def nomination_autoreview_group(self, ctx: Context) -> None:
+        """Commands for enabling or disabling autoreview."""
+        await ctx.send_help(ctx.command)
+
+    @nomination_autoreview_group.command(name='on')
+    @has_any_role(Roles.admins)
+    async def autoreview_on(self, ctx: Context) -> None:
+        """
+        Turn on automatic posting of reviews.
+
+        This will post reviews up to one day overdue. Older nominations can be
+        manually reviewed with `!tp post_review <user_id>`.
+        """
+        await self.talentpool_settings.set(AUTOREVIEW_ENABLED_KEY, True)
+        await self.reviewer.reschedule_reviews()
+        await ctx.send(':white_check_mark: Autoreview turned on')
+
+    @nomination_autoreview_group.command(name='off')
+    @has_any_role(Roles.admins)
+    async def autoreview_off(self, ctx: Context) -> None:
+        """Turn off automatic posting of reviews."""
+        await self.talentpool_settings.set(AUTOREVIEW_ENABLED_KEY, False)
+        self.reviewer.cancel_all()
+        await ctx.send(':white_check_mark: Autoreview turned off')
+
+    @nomination_autoreview_group.command(name='status')
+    @has_any_role(*MODERATION_ROLES)
+    async def autoreview_status(self, ctx: Context) -> None:
```

Comment:
It could be, although I think both ways are equally fine. The current method is similar to how `!defcon` does it, and as it's a command that wont be used very often I think having it show the help by default may be helpful.